### ISSUE TITLE
WCAG: e-mail field label; link ID

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2176]
page: https://stadjerspas.gemeente.groningen.nl/start

The input field where the e-mail address must be entered has no label associated with this input field in the code.

The reading software cannot tell its user what information this input field accepts.

In the code, the fields must be linked to labels via an ID, so that it is clear to auxiliary software what needs to be entered where.

![Screenshot 2021-03-19 at 11 19 19](https://user-images.githubusercontent.com/33207036/111765836-219f4700-88a5-11eb-8c81-80db6742b08c.png)

**Comment by GerbenBosschieter** [comment]
Checked and passed on Release/v0.24.0
